users/bescka/cuba chups.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <link rel="Stylesheet" type="text/css" href="style.css" />
+  <title>Flounders and Codes - Chupa chups</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+  <link rel="icon" href="/assets/fc_favicon_96_2.ico" type="image/x-icon">
+  <link rel="shortcut icon" href="/assets/fc_favicon_96_2.ico" type="image/x-icon">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <div class="container">
+    <div class="ascii-art">
+      - ~- ~ - - O o <br>
+      &nbsp; ~- - ~ - o -<br>
+      ~ - ><(((º> ~ -<br>
+        &nbsp; - ~ - - -- ~ <br>
+    </div>
+    <span class="title_container">
```

Comment:
You could make the page header into a link back to the home page

Reply:
Good idea! Implemented.